### PR TITLE
add progress dialog for export/remove

### PR DESCRIPTION
**Pull request by hawkeyexp**
Add dialog with progress bar to make it visible the addon is working for you :-)

screenshot000
screenshot001

### CODE REVIEW

File: resources/lib/Library.py [hunk 1]
```diff
@@ -250,14 +254,21 @@ def add_movie (self, title, alt_title, year, video_id, build_url):
...
         filename = os.path.join(dirname, movie_meta + '.strm')
+        progress=xbmcgui.DialogProgress()
+        progress.create(kodi_helper.get_local_string(650),movie_meta)
         if xbmcvfs.exists(filename):
             return
         if not xbmcvfs.exists(dirname):
             xbmcvfs.mkdirs(dirname)
         if self.movie_exists(title=title, year=year) == False:
+            progress.update(50)
+            time.sleep(0.5)
```
Comment (asciidisco) [inline resources/lib/Library.py]: Just curious, why are the `sleep` statements needed?

**Comment by asciidisco** [comment]
Looks good to me, just added a tiny comment for my own understanding.

File: resources/lib/Library.py (hunk 1 above)
Comment (hawkeyexp) [reply resources/lib/Library.py]: Movies are only a single file which is written so it jumps nearly instant to 100% in dialog so it is only a soft break for better visibility.

**Comment by asciidisco** [comment]
Looks good to me then :)

**Pull request merged**